build/scripts/storage.js

```diff
@@ -29,11 +29,21 @@ loads them if they have been
 function Initialize() {
 	chrome.storage.sync.get('accomplishGoalsList', function(response){
 		if (typeof response.accomplishGoalsList !== 'undefined') {
+			var index = 0;
 			response.accomplishGoalsList.forEach(function(goal) {
-				$(document.body).append('<p> Title: ' + goal.title + '</p>');
-				$(document.body).append('<p> Begin: ' + goal.begin + '</p>');
-				$(document.body).append('<p> End: ' + goal.end + '</p>');
-				$(document.body).append('<p> Description: ' + goal.description + '</p>');
+				var newGoal = 
+				'<div class=goal-item index=' + index + '>' +
```

Comment:
Can we append the parent node, and then append the children? Appending parent node should return that DOM element, so we can save it to a variable and append to the variable.